### ISSUE TITLE
LC0068 - False Positive - Record Identical to PageExtension Record

### ISSUE THREAD

**Issue by ans-bar-bm** [StefanMaron/BusinessCentral.LinterCop#942]
Pretty much the same as #728, just with a Page Extension instead of a Page.
I'd say the same logic should apply here, if the user can already access the page then reads of the same table should not raise a missing permissions diagnostic.

```al
pageextension 50000 MyCustomerList extends "Customer List"
{
    trigger OnAfterGetRecord()
    var
        Customer2: Record Customer;
    begin
        Customer2.FindFirst(); // LC0068 is raised here
    end;
} 
```

**Comment by Arthurvdv** [comment]
Awesome work, thanks a ton for helping out with this!
